core/domain/summary_services.py

```diff
@@ -74,16 +90,29 @@ def get_human_readable_contributors_summary(contributors_summary):
 def get_learner_collection_dict_by_id(
         collection_id, user_id, strict=True, allow_invalid_explorations=False,
         version=None):
-    """Creates and returns a dictionary representation of a collection given by
-    the provided collection ID. This dictionary contains extra information
-    along with the dict returned by collection_domain.Collection.to_dict()
-    which includes useful data for the collection learner view. The information
-    includes progress in the collection, information about explorations
-    referenced within the collection, and a slightly nicer data structure for
-    frontend work.
-    This raises a ValidationError if the collection retrieved using the given
-    ID references non-existent explorations.
-    which includes useful data for the collection learner view.
+    """Get a dictionary representation of a collection given by the provided
+    collection ID.
```

Comment:
Add sentence at end: This dict includes user-specific playthrough information.

(This provides motivation for why there's a "learner" in the function name and why we're not just returning a "standard" collection dict.)